components/monitoring/base/prometheus-servicemonitors.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  namespace: appstudio-workload-monitoring
```

Comment:
As discussed in a slack thread - ServiceMonitors should be ideally stored in a component namespace:
https://coreos.slack.com/archives/CHK0J6HT6/p1670234963843059?thread_ts=1669991703.870609&cid=CHK0J6HT6